src/pages/aura.f7.html

```diff
@@ -55,11 +61,14 @@
         });
       },
       resizeMessagesContent: function (){
+        let logo_aura = Dom7('#logo_aura');
         let aura = Dom7('#tab_aura');
         let messagebar = Dom7('.messagebar');
         let messagesContent = Dom7('.messages-content');
         let height = aura.height() - messagebar.height() +'px';
```

Comment:
```suggestion
        let height = aura.height() - messagebar.height() + 'px';
```

Espaço entre operadores para melhorar a legibilidade.